### PR TITLE
Promotion Page

### LINKED ISSUE #234: [Template] Build out Promotion Page
Create template for promotion pages. eg. https://www.hubblehomes.com/promotions/promotions-detail/your-way-savings
https://www.hubblehomes.com/promotions/promotions-detail/Hubble-Homes-for-Heroes

### PR DESCRIPTION
**Pull request by varunmitra**
Fix #234 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/promotions/promotions-detail/your-way-savings
- After: https://issue-234--hubblehomes-com--aemsites.hlx.live/promotions/promotions-detail/your-way-savings

### CODE REVIEW

File: templates/promotion/promotion.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+/* eslint-disable function-paren-newline, object-curly-newline */
+import { div, a } from '../../scripts/dom-helpers.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main > .section');
+  const divPromotion = div({ class: 'promotion-section' });
+  const leftSection = doc.querySelector('.left-column');
+  const rightSection = doc.querySelector('.right-column');
+  leftSection.style.display = 'block';
+  rightSection.style.display = 'block';
```
Comment (bhellema) [inline templates/promotion/promotion.js:12]: Can this not go in the style sheet?

File: templates/promotion/promotion.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable function-call-argument-newline */
+/* eslint-disable max-len */
+/* eslint-disable function-paren-newline, object-curly-newline */
+import { div, a } from '../../scripts/dom-helpers.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main > .section');
+  const divPromotion = div({ class: 'promotion-section' });
+  const leftSection = doc.querySelector('.left-column');
```
Comment (bhellema) [inline templates/promotion/promotion.js:9]: The left-column has a section that's adding additional padding.  Need to figure out why the section is being added here, or we can blow it away.

```
leftSection.classList.remove('section')
```


The result of the previous two updates will align the text with the breadcrumb.
image

vs
image

File: templates/promotion/promotion.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable function-call-argument-newline */
+/* eslint-disable max-len */
+/* eslint-disable function-paren-newline, object-curly-newline */
+import { div, a } from '../../scripts/dom-helpers.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main > .section');
+  const divPromotion = div({ class: 'promotion-section' });
```
Comment (bhellema) [inline templates/promotion/promotion.js:8]: Wrap the promotion-section in a `section`.

```
const divPromotion = div( { class: 'section' }, div({class: 'promotion-section'));
```

File: templates/promotion/promotion.js (hunk 1 above)
Comment (varunmitra) [reply templates/promotion/promotion.js:12]: somehow, `style = display:none` gets added to left and right column. This has to be overwritten in code for the section to be displayed. 
Screen Shot 2024-07-25 at 12 16 49 PM

File: blocks/heading/heading.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+.heading>em {
```
Comment (bhellema) [inline blocks/heading/heading.css:9]: This could be:
```
.heading > em,
.heading > p {
   font-size: var(--body-font-size-s);
}
```

**Review by bhellema — APPROVED** [review]

**Pull request merged**